.gitignore

```diff
@@ -4,7 +4,6 @@
 *.pdb
 *.suo
 *.js
-*.cjs
```

Comment:
Probably best to keep this, so nobody accidentally checks in random .cjs files.

Reply:
- [x] I've removed this hunk.